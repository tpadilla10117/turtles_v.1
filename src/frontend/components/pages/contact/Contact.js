import React from 'react';
import {
  Footer
} from '../../utils.js';

function Contact() {
  return (
    <div>
        Greetings from contact
        <Footer />
    </div>
  )
}

export default Contact;