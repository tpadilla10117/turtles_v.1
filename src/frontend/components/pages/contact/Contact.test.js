import { shallow } from 'enzyme';
import React from 'react';
import Contact from './Contact';

describe('The Contact.js component:', () => {
    it('renders the Contact.js component', () => {
        expect.assertions(2);
        expect(shallow(<Contact />).length).toEqual(1);
        expect(shallow(<Contact />)).toMatchSnapshot();
    })
})