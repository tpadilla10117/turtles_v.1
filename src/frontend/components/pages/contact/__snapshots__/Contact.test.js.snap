// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Contact.js component: renders the Contact.js component 1`] = `ShallowWrapper {}`;
